XamarinCommunityToolkit/Core/CameraMediaSource.shared.cs

```diff
@@ -6,7 +6,6 @@
 
 namespace Xamarin.CommunityToolkit.Core
 {
-	// TODO: Check if we can do these changes in Xamarin.Forms to not have our own version of the MediaSource
 	// TODO 2: Is this even used?! Not for iOS it seems...
 	public class CameraMediaSource : MediaSource
```

Comment:
I guess we can/should drop this file. It is not used

Reply:
removed